docker/cloudbuild.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+# Build the main getsentry/sentry image
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '--build-arg', 'SOURCE_COMMIT=$COMMIT_SHA', '-t', 'gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA', '-t', 'gcr.io/$PROJECT_ID/sentry:$SHORT_SHA', '-t', 'gcr.io/$PROJECT_ID/sentry:latest', '-f', './docker/Dockerfile', '.' ]
+# Derive the -onbuild variant
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA-onbuild', '-t', 'gcr.io/$PROJECT_ID/sentry:$SHORT_SHA-onbuild', '-t', 'gcr.io/$PROJECT_ID/sentry:latest-onbuild', '-f', './docker/onbuild/Dockerfile', '.' ]
+# Smoke tests
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['config', 'get', 'system.secret-key']
```

Comment:
I have a low level of confidence this will actually work, since nothing is directing it to use the image we just built. Unless this behavior works implicitly, I'd expect this to translate into a call such as `docker config get system.secret-key` inside the `docker` container.

Reply:
It was my mistake, just check the very latest version where I tell it to use the image we just built.